webapp/src/Requests/Player/Player.ts

```diff
@@ -0,0 +1,16 @@
+// TODO: Move independent service functions to this class. Will do when we have the store fixed
+export class PlayerService {
+    private static instance: PlayerService
+    private constructor() {}
+
+    public static getInstance() {
+        if (this.instance == null) {
+            this.instance = new PlayerService()
+        }
+        return this.instance
+    }
+
+    public shutTheLinterUp() {
```

Comment:
Add a comment as to why his is necessary?

Reply:
See line 1

Reply:
Addressed in b44dc6591be55e284c158783d5a39b90d885e350

Reply:
Addressed in b44dc6591be55e284c158783d5a39b90d885e350